.github/workflows/cicd.yaml

```diff
@@ -14,6 +14,7 @@ jobs:
         fetch-depth: 0
     - run: ./bin/make ci
     - run: ./bin/make firebase-deploy
+      if: github.event_name == 'pull_request' # only run on PR, deploy prod/master in release job
```

Comment:
oops shouldn't have approved. needs `${{ ... }}` I think

Reply:
ok, I will add it down below too ( it doesn't really need it it seems.)

Reply:
ok, I will add it down below too ( it doesn't really need it it seems.)